h/static/scripts/guest.coffee

```diff
@@ -242,6 +238,11 @@ module.exports = class Annotator.Guest extends Annotator
     this.plugins.CrossFrame.sync([annotation])
     annotation
 
+  createHighlight: ->
+    annotation = super
```

Comment:
No need to have a separate upstream method at all, as explained above. Simply include "super" here.